src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -61,6 +61,13 @@ public static void ComputeActiveProcessList(IReadOnlyList<IPositionedOrganelle>
     {
         result ??= new List<TweakedProcess>();
 
+        var speedMultipliers = new Dictionary<BioProcess, float>();
```

Comment:
This code should be rewritten to not use this extra temporary memory.

Reply:
Should the Dictionary be stored outside of the method's domain?

Reply:
Yes, if there's truly no way to write the logic in a way that doesn't require this store first and then apply approach.